docs/specs/git.yml

```diff
@@ -408,55 +408,20 @@ outputs:
       "gitcommit": "https://github.com/testowner/testrepo/blob/e662dbc7c5e582a14b863d6a6a82e495d6272f91/docs/index.md",
     }
 ---
-# Show git contribution info for vsts repo
+# Show (original) content git (branch/commit) url for vsts
 repos:
-  https://docascode.visualstudio.com/contribution/_git/a:
```

Comment:
>git-url [](start = 37, length = 7)

name's a bit confusing. git-url here refers to an `org/group`, while in the contribution config, it becomes a `repo name`

Reply:
it's just to make the url different with other cases, no real meaning here